src/main/java/lotto/Validator/InputValidator.java

```diff
@@ -0,0 +1,46 @@
+package lotto.Validator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class InputValidator {
+
+    private static final String SEPARATOR = ",";
+    private static final int MONEY_MIN_AMOUNT = 1000;
+    private static final int MONEY_MAX_AMOUNT = 21 * 100000000;
```

Comment:
int의 최대값 범위 21억을 표현하셨다면
```suggestion
private static final int MONEY_MAX_AMOUNT = Integer.MAX_VALUE;
```
표현하는건 어떨까요?